core/local/analysis.js

```diff
@@ -36,34 +36,12 @@ module.exports = function analysis (events /*: LocalEvent[] */, pendingChanges /
   return separatePendingChanges(changes, pendingChanges)
 }
 
-const panic = (context, description) => {
-  log.error(_.merge({sentry: true}, context), description)
-  throw new Error(description)
-}
-
 function analyseEvents (events /*: LocalEvent[] */, pendingChanges /*: LocalChange[] */) /*: LocalChange[] */ {
   const stopMeasure = measureTime('LocalWatcher#analyseEvents')
-  // OPTIMIZE: new Array(events.length)
-  const changes /*: LocalChange[] */ = []
```

Comment:
I wonder why we were listing all changes separately... Isn't it impossible to end up with multiple changes for the same inode? Would it even be possible to have multiple changes for the same (destination) path?